Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM letsencrypt/boulder-tools:2016-07-05
+FROM letsencrypt/boulder-tools:2016-07-08
```

Comment:
Add a comment: "To minimize fetches, this should match the tag used for the HSM in docker-compose.yml"